### PR TITLE
[REPLICATED-1699] filters for multi-scheduler config option support

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: config_group.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ type ConfigGroup struct {
 	Title       string        `yaml:"title" json:"title"`
 	Description string        `yaml:"description" json:"description"`
 	TestProc    *TestProc     `yaml:"test_proc" json:"test_proc"`
-	Items       []*ConfigItem `yaml:"items" json:"items" validate:"dive,exists"`
 	When        string        `yaml:"when" json:"when"`
+	Filters     []string      `yaml:"filters" json:"filters" validate:"dive,exists"`
```
Comment (divolgin) [inline config_group.go:9]: How about `SchedulerFilters`?  Is that the intention?

File: config_item.go [hunk 2]
```diff
@@ -22,6 +22,7 @@ type ConfigItem struct {
 	Required              bool                   `yaml:"required" json:"required"`
 	TestProc              *TestProc              `yaml:"test_proc" json:"test_proc"`
 	IsExcludedFromSupport bool                   `yaml:"is_excluded_from_support" json:"is_excluded_from_support"`
+	Filters               []string               `yaml:"filters" json:"filters" validate:"dive,exists"`
```
Comment (divolgin) [inline config_item.go:25]: Same here

**Review by divolgin — APPROVED** [review]

File: config_group.go (hunk 1 above)
Comment (emosbaugh) [reply config_group.go:9]: i made it ambiguous on purpose

Comment (emosbaugh) [reply config_group.go:9]: example

``` yaml
filters:
- scheduler=kubernetes
- some_other=abcd
```

**Pull request merged**